Household energy consumption, by type of dwelling.ipynb

```diff
@@ -0,0 +1,316 @@
+    "\n",
+    "import pandas as pd\n",
+    "import os\n",
+    "import qgrid\n",
+    "import ipywidgets\n",
+    "import numpy as np"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 51,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2020-12-07T03:03:36.304227Z",
+     "start_time": "2020-12-07T03:03:36.291715Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "file_id = \"25100061\"\n",
+    "filepath = \"data/{}_MetaData-dirty.df\".format(file_id)\n",
+    "df_dictionary = pd.read_pickle('data/{}_MetaData-dirty.df'.format(file_id))\n",
+    "df = pd.read_pickle('data/{}-dirty.df'.format(file_id))\n",
+    "qgrid.enable()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "###### Guide to StatsCan datasets\n",
+    "See https://www.statcan.gc.ca/eng/developers/csv/user-guide for description of column names.  Re-summarized here for reference:\n",
+    "\n",
+    "Comments on columns:\n",
+    "\n",
+    "|Column | StatsCan Notes | Our Notes |\n",
+    "|:--| :--| :--|\n",
+    "|UOM ID (Units of measure ID) | | seems only to be necessarily to have a unique identifier to deal with Eng/French translation of units of measurement, so ignore this for now|\n",
+    "|SCALAR_FACTOR | The scalar factor associated with a data series, displayed as text. | Disregard this if all entries are 'units' (scalar factor of 1)|\n",
+    "| Scalar_ID | | See above |\n",
+    "| Vector |Unique variable length reference code time-series identifier, consisting of the letter 'V', followed by up to 10 digits. (i.e. V1234567890, V1, etc.) | Don't understand this | \n",
+    "| Status | Shows various states of a data value using symbols. These symbols are described in the [symbol legend](https://www.statcan.gc.ca/eng/concepts/definitions/guide-symbol) and notes contained in the metadata file. Some symbols accompany a data value while others replace a data value. i.e. – A, B, C, D, E, F,.., X, 0s | It's at row 98 of the data dictionary here|\n",
+    "| Symbol | Describes data points that are preliminary or revised, displayed using the symbols p and r. These symbols accompany a data value.| Can drop this | \n",
+    "| DECIMALS | This field displays the decimal precision for a given value. | We should append this to VALUE before converting it to numeric |\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 52,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "bd366ab858344a6daacd3d61adba36eb",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "QgridWidget(grid_options={'fullWidthRows': True, 'syncColumnCellResize': True, 'forceFitColumns': True, 'defau…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "symbol_legend = df_dictionary.iloc[0:1].copy()\n",
```

Comment:
Symbol legend is at a different point in each datasets dictionary.  I think it's [33:50] here.

Reply:
Took the liberty to make this change, hope that's alright.